fetch-utils.js

```diff
@@ -0,0 +1,19 @@
+const SUPABASE_URL = 'https://sushgnkqkgfdkwxudpdy.supabase.co';
+const SUPABASE_KEY =
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InN1c2hnbmtxa2dmZGt3eHVkcGR5Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2NjAwNjEyMDEsImV4cCI6MTk3NTYzNzIwMX0.PWaLx9CyI6jaOzBx-1JPnId6_IrMlC4rYSEFZtsLwPw';
+
+const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
+
+export async function createPoll(poll) {
+    const response = await client.from('polls').insert({ question: poll.question, answer1: poll.answer1, answer2: poll.answer2, votes1: poll.votes1, votes2: poll.votes2 });
```

Comment:
Another way to do this is to send an entire "poll" into insert. It has the same functionality as what you're doing here, but I would say is slightly more common.